### PR TITLE
bump utils to fix slow notification rendering

### PR DESCRIPTION
**Pull request by leohemsted**

### CODE REVIEW

**Comment by quis** [comment]
https://github.com/alphagov/notifications-utils/pull/511

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .DEFAULT_GOAL := help
 SHELL := /bin/bash
-DATE = $(shell date +%Y-%m-%d:%H:%M:%S)
+DATE = $(shell date +%Y-%m-%dT%H:%M:%S)
```
Comment (tombye) [inline Makefile]: Can this be split off into its own commit?
